marlowe-playground-client/src/Simulator/State.purs

```diff
@@ -216,7 +216,7 @@ applyPendingInputs oldState@{ executionState: SimulationRunning executionState }
 
         newExecutionState =
           ( set _transactionError Nothing
-              <<< set _transactionWarnings (fromFoldable txOutWarnings)
+              <<< over _transactionWarnings (flip append $ fromFoldable txOutWarnings)
```

Comment:
I changed from set to over because if you made a step without looking at the warning you'd lose it